pkg/setup.py

```diff
@@ -32,6 +38,20 @@ def getreqs():
         reqs = f.readlines()
         return [x.strip() for x in reqs]
 
+def getdatafiles():
+    data_files = []
+
+    # Man files
+    man_sections = {}
+    for file in listdir(mandir):
+        print('Found man page: %s' % file)
```

Comment:
Also, prefer consistent string formatting (avoid `%` formatting in favour of `.format()` formatting) for all current and future contributions, `CONTRIBUTING.md` will be updated to reflect this and a few other changes

Reply:
For my own ignorance, why is `.format()` viewed as superior?

Reply:
I don't know about *superior*... it is the standard that was introduced to eventually replace `%` formatting, though I suspect that `%` formatting will stick around for quite some time...
Both have a few advantages, though I think `.format()` has more.
The primary thing here (in this comment) is maintaining consistency